APITaxi/api/taxi.py

```diff
@@ -114,106 +113,43 @@ def put(self, taxi_id):
 
 @ns_taxis.route('/', endpoint="taxi_list")
 class Taxis(Resource):
-    def filter_zone(self, taxi, p):
-        if not taxi:
-            current_app.logger.debug('Taxi {} not fount in db')
-            return False
-        taxi = taxi[0]
-        zupc_id = taxi['ads_zupc_id']
-        if not zupc_id in list(self.zupc_customer.keys()):
-            current_app.logger.debug('Taxi {} not in customer\'s zone'.format(
-                taxi.get('taxi_id', 'no id')))
-            return False
-        if not self.zupc_customer[self.parent_zupc[zupc_id]].contains(
-                        Point(float(p[0]), float(p[1]))):
-            current_app.logger.debug('Taxi {} is not in its zone'.format(
-                taxi.get('taxi_id', 'no id')))
-            return False
-        return True
 
-    def set_not_available(self, lon, lat, name_redis, radius):
-        store_key = name_redis+'_operateur'
-        g.keys_to_delete.append(store_key)
-        redis_store.georadius(current_app.config['REDIS_GEOINDEX'], lon, lat, radius, 'm',
-                              store_dist=store_key)
-        redis_store.zinterstore(store_key, [store_key,
-                                current_app.config['REDIS_TIMESTAMPS'],
-                                current_app.config['REDIS_NOT_AVAILABLE']])
-        self.not_available = {t[0].decode().split(':')[0] for t
-                              in redis_store.zscan_iter(store_key)}
-
-    def check_freshness(self):
+    @staticmethod
+    def check_freshness():
```

Comment:
I think the function name doesn't describe what it is actually done. It does more than checking freshness, since a task is emitted under certain conditions.